Home_Work_4/task1.py

```diff
@@ -0,0 +1,28 @@
+# Напишите функцию для транспонирования матрицы
+
+import numpy
+
+
+def print_matrix(matrix_a):
+    for i in range(len(matrix_a)):
+        for j in range(len(matrix_a[i])):
+            print(matrix_a[i][j], end=' ')
+        print()
+
+
+def reverse_matrix(matrix_b):
+    new_matrix = list(zip(*matrix_b[::]))
+    return new_matrix
```

Comment:
Этот вариант мне нравится больше всего: быстро, просто, изящно. Хотя не совсем понятно, зачем вам срез матрицы. Кроме того, на выходе вы получаете список кортежей, а не список списков. Но это легко исправляется.
```suggestion
    return [*map(list, zip(*matrix_b)]
```